news/2017-07-24-scorewatch-july-week-3.md

```diff
@@ -0,0 +1,49 @@
+tumblr_url: http://osunews.tumblr.com/post/162432090863/scorewatch-july-week-1
+---
+
+A calmer week, but that does not stop a few from pulling off SS plays, impressive osu!taiko and osu!catch plays for this week’s Scorewatch!
+
+<!-- INSERT BANNER HERE HTML COMMENTS HAHAHA -->
+
+[Megure](https://osu.ppy.sh/u/3468264) unexpectedly got the first ever SS on the Nao’s Eclipse difficulty on [The Koxx - A FOOL MOON NIGHT](https://osu.ppy.sh/b/1141084), beating out the likes of Dustice and Musty. What control does this guy have in his bones?
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/KptQoF9mGeA" frameborder="0" allowfullscreen></iframe></br>
+
+Remember [Recia](https://osu.ppy.sh/u/1787027)? Well, he has gone and set the No. 3 score on [Camellia - Light it up](https://osu.ppy.sh/b/1093078?m=0) with an impressive 99.32% HDHR FC to earn 573pp, beating out Toy, Azer and Adamqs. Not too bad for the No. 1 Japanese.
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/S4IRUC5E7RY" frameborder="0" allowfullscreen></iframe></br>
+
+[WubWoofWolf](https://osu.ppy.sh/u/39828) picks a spot in this article with a massive HDHR SS on [DJ S3RL - T-T-Techno (feat. Jesskah)](https://osu.ppy.sh/s/83560) to take 349pp and the No. 1 spot. Weird spacings and sliders prove that this map would take an amazing play for it to get SSed, and the former No. 1 has shown his skill to do so.
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/aKuLWtL-qNw" frameborder="0" allowfullscreen></iframe></br>
+
+[_yu68](https://osu.ppy.sh/u/6170507) pushes his limits further by beating out [shinchikuhome](https://osu.ppy.sh/u/3174184) on [DragonForce - Cry Thunder](https://osu.ppy.sh/b/1065704) with a crazy 99.90% DT FC to earn 623pp and sniping him for the No. 1 spot. What else is there to say?
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/2L_1pHPLq7g" frameborder="0" allowfullscreen></iframe></br>
+
+[Spectator](https://osu.ppy.sh/u/702598) comfortably took [goreshit - burn this moment into the retina of my eye](https://osu.ppy.sh/s/481632) and tore it into shreds with a 99.76% HD FC to take 947pp and the No. 1 spot. Have a look at 217bpm fruit catching action!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/Ai4aSRSp0Rg" frameborder="0" allowfullscreen></iframe></br>
+
+NOTABLE MENTIONS
+-----
+
+[kablaze](https://osu.ppy.sh/u/3043603) returns to Scorewatch with a very strong play on [cillia - Fairytale](https://osu.ppy.sh/b/759192?m=0), scoring an amazing 99.79% HDHR FC to earn 471pp and the No. 1 score on the map (not forgetting to mention his 66.66 UR!). Talk about being a consistent player, this play is not to be underestimated!
+
+Here he is once again. [kablaze](https://osu.ppy.sh/u/3043603)’s HDHR SS on [RYO - Shuffle Heaven](https://osu.ppy.sh/b/235470) is pretty extraordinary, racking up 460pp and the No. 1 spot. What a way to snipe WubWoofWolf!
```

Comment:
Replace "here he is once again" with something like "In all of his extraordinary glory, kablaze does it again with a HDHR SS.." etc